src/conversion/mapper.js

```diff
@@ -159,13 +159,12 @@ export default class Mapper {
 	toModelPosition( viewPosition ) {
 		const data = {
 			viewPosition: viewPosition,
-			modelPosition: this._defaultToModelPosition( viewPosition ),
 			mapper: this
 		};
 
 		this.fire( 'viewToModelPosition', data );
 
-		return data.modelPosition;
+		return data.modelPosition ? data.modelPosition : this._defaultToModelPosition( viewPosition );
```

Comment:
> I wanted to say in the ticket that I remember that it was already possible to add the custom listener before the default mapping and I don't know why it's not possible anymore, but I thought it was a dream ;)

Yes, it was like this for a while but then it got changed. I don't remember the exact reasons, though. I can change it so the default mapping callback is fired with `low` priority. Not a big deal.

> Could you link where are these tests, because I may miss something.

I meant these:
https://github.com/ckeditor/ckeditor5-engine/blob/master/tests/conversion/mapper.js#L209-L223
https://github.com/ckeditor/ckeditor5-engine/blob/master/tests/conversion/mapper.js#L282-L296

I haven't added new tests because, from API point of view, in fact nothing changed, other than the fact that you don't get "default" position in the callback.